policyengine_us/parameters/gov/states/nj/tax/income/exclusions/retirement/income_limit.yaml

```diff
@@ -0,0 +1,15 @@
+description: New Jersey filers above this income are not eligible for pension/retirement income exclusion and other retirement exclusion.
+metadata:
+  period: year
+  type: currency-USD
+  label: New Jersey pension/retirement retirement income exclusion and other retirement exclusion limit.
```

Comment:
```suggestion
  label: New Jersey pension/retirement retirement income exclusion limit.
```

Reply:
I think we can just use pension/retirement retirement income exclusion without the second part in all yaml files as it is more concise